site/content/user-guide/amqp-adapter.md

```diff
@@ -262,6 +261,36 @@ The table below provides an overview of the properties that must be set on a com
 | *correlation-id* | yes             | *properties*             | *string*     | MUST contain the value of the *correlation-id* property of the command request message. |
 | *status*         | yes             | *application-properties* | *integer*    | MUST contain a status code indicating the outcome of processing the command at the device (see [Command & Control API]({{< ref "/api/Command-And-Control-API.md" >}}) for details). |
 
+### Examples
+
+Using the `amqp-command` spring profile, the AMQP Command-line client can be used to receive and send a command response message from/to an application. The client is used to simulate a device to receive commands and for sending command responses back to the application. The command CLI client is used to simulate an application sending commands to devices and receiving command responses from devices.
```

Comment:
> The AMQP adapter client can be used to simulate a device which receives commands and sends responses back to the application. The command line client is used to simulate an application sending commands to devices and receiving command responses from devices.

Reply:
> The command line client is

dont't you think this could be confused with the `terminal` itself? how about the `command CLI` ?